Assets/HoloToolkit.Unity.SpectatorView.meta

```diff
@@ -0,0 +1,3 @@
+﻿fileFormatVersion: 2
```

Comment:
This folder should live under the HoloToolkit folder, probably in the SpectatorView folder, instead of being in a new top-level folder.

![image](https://user-images.githubusercontent.com/3580640/39490092-7f3e59d4-4d3c-11e8-9b9d-3df30a6d93b5.png)